autoload/auto_git_diff.vim

```diff
@@ -70,6 +68,25 @@ function! s:show_git_diff_impl(hash, vertsplit, opts) abort
     silent wincmd p
 endfunction
 
+function! s:get_git_diff(hash, opts) abort
+    let prefix = has("win32") ? "set LANG=C & " : "LANG=C "
+
+    let diff_command = "git diff ".a:opts." ".a:hash."~1 ".a:hash
+    silent let out = system(prefix.diff_command)
```

Comment:
It seems that nothing will be shown if `git diff` command fails here for some reason except for the first commit.

Please save the contents of `out` and return it if the next execution of `git diff` is failed.

Reply:
Right, fixed in 304176d.